sway-fmt/src/traversal_helper.rs

```diff
@@ -278,4 +407,38 @@ mod tests {
             "a::{bar, foo};"
         );
     }
+
+    #[test]
+    fn test_format_use_statement_length_leaves_input_unchanged() {
+        let s = "a::b::{c, d::{self, f}};";
+        assert_eq!(format_use_statement_length(s, 100, 0), s);
+    }
+
+    #[test]
+    fn test_format_use_statement_length_formats_long_input() {
+        let s = "std::{address::*, assert::assert, block::*, chain::auth::*, context::{*,text::{call_frames::*, dial_frames::{Transaction, TransactionParameters}, token_storage::{CallData, Parameters}}}, contract_id::ContractId, hash::*, panic::panic, storage::*, token::*};";
+        let expected = format!(
+            r#"{ALREADY_FORMATTED_LINE_PATTERN}std::{{
+{ALREADY_FORMATTED_LINE_PATTERN}    address::*,
+{ALREADY_FORMATTED_LINE_PATTERN}    assert::assert,
+{ALREADY_FORMATTED_LINE_PATTERN}    block::*,
+{ALREADY_FORMATTED_LINE_PATTERN}    chain::auth::*,
+{ALREADY_FORMATTED_LINE_PATTERN}    context::{{
+{ALREADY_FORMATTED_LINE_PATTERN}        *,
+{ALREADY_FORMATTED_LINE_PATTERN}        text::{{
+{ALREADY_FORMATTED_LINE_PATTERN}            call_frames::*,
+{ALREADY_FORMATTED_LINE_PATTERN}            dial_frames::{{Transaction, TransactionParameters}},
+{ALREADY_FORMATTED_LINE_PATTERN}            token_storage::{{CallData, Parameters,
+{ALREADY_FORMATTED_LINE_PATTERN}        }}}}
```

Comment:
Hmmm. Should this not be
```suggestion
{ALREADY_FORMATTED_LINE_PATTERN}            token_storage::{{CallData, Parameters}},
{ALREADY_FORMATTED_LINE_PATTERN}        }}
```

Reply:
@adlerjohn You are correct (oversight on my part). Just pushed an update